<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/back.png);
background-repeat: no-repeat;
background-position: center center;
background-attachment: fixed;
</string>
  </property>
  <widget class="QPushButton" name="btnStartGame">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;NiseSegaSonic&quot;;
QLabel{background-color: grey}
QLineEdit {background-color: rgb(188, 188, 188)}</string>
   </property>
   <property name="text">
    <string>Start Game</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>470</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;NiseSegaSonic&quot;;</string>
   </property>
   <property name="text">
    <string>Audio?</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
